HomeWork_Kotlin/app/src/main/java/com/gmail/vladkhinski/homework_kotlin/CalculatorFragment.kt

```diff
@@ -0,0 +1,148 @@
+package com.gmail.vladkhinski.homework_kotlin
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.gmail.vladkhinski.homework_kotlin.databinding.FragmentCalculatorBinding
+import net.objecthunter.exp4j.ExpressionBuilder
+
+class CalculatorFragment : Fragment() {
+
+    private var _binding: FragmentCalculatorBinding? = null
+    private val binding get() = requireNotNull(_binding) { "View was Destroyed" }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        return FragmentCalculatorBinding.inflate(inflater, container, false)
+            .also { _binding = it }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+            btn0.setOnClickListener {
+                val str = "0"
+                mainView.append(str)
```

Comment:
можно сразу передавать без объявления отдельных переменных 
```suggestion
                mainView.append("0")
```